source/base/parameter_handler.cc

```diff
@@ -2622,4 +2676,35 @@ MultipleParameterLoop::Entry::memory_consumption() const
           sizeof(type));
 }
 
+void
+parse_input(std::string const parameter_file,
```

Comment:
separate patch, as mentioned above

Reply:
I would like to have the discussion here because it is related to the design questions raised above, i.e., this function calls the ``assert_()`` function. We could then still merge as a separate PR?